vllm/config.py

```diff
@@ -242,6 +269,19 @@ def _init_multimodal_config(
 
         return None
 
+    def _init_pooler_config(
+            self, pooling_type, pooling_norm, pooling_softmax,
+            pooling_step_tag_id,
+            pooling_returned_token_ids) -> Optional["PoolerConfig"]:
```

Comment:
Can you add type annotations to this method?

Reply:
I have made the changes as you recommended.